messaging.rst

```diff
@@ -175,39 +58,58 @@ Invariants
+- The transfer is valid only if the ``locked_amount`` is smaller than the sender's :term:`capacity`.
 
 Fields
 ^^^^^^
 
-+----------------------+---------------+------------------------------------------------------------+
-| Field Name           | Field Type    |  Description                                               |
-+======================+===============+============================================================+
-|  lock                | HashTimeLock  | The lock for this mediated transfer                        |
-+----------------------+---------------+------------------------------------------------------------+
-|  balance_proof       | BalanceProof  | Balance proof for this transfer                            |
-+----------------------+---------------+------------------------------------------------------------+
-|  initiator           | address       | Initiator of the transfer and person who knows the secret  |
-+----------------------+---------------+------------------------------------------------------------+
-|  target              | address       | Final target for this transfer                             |
-+----------------------+---------------+------------------------------------------------------------+
+The same as in a :ref:`direct transfer <direct-transfer-message>`, and additionally:
+
++----------------------+---------------+-------------------------------------------------------------+
+| Field Name           | Field Type    |  Description                                                |
++======================+===============+=============================================================+
+|  lock                | HashTimeLock  | The lock for this mediated transfer                         |
++----------------------+---------------+-------------------------------------------------------------+
+|  locked_amount       | integer       | Amount to be transferred including fees                     |
++----------------------+---------------+-------------------------------------------------------------+
+|  fee                 | integer       | Remaining amount to be used as transfer fees                |
++----------------------+---------------+-------------------------------------------------------------+
+|  target              | address       | Final target for this transfer                              |
++----------------------+---------------+-------------------------------------------------------------+
+|  secrethash          | hash          | Hash of the secret that will unlock the transfer            |
++----------------------+---------------+-------------------------------------------------------------+
 
+Fees are always payable by the initiator. The intermediary node receiving this can deduct its own fee and lower ``fee`` to the remaining fee. It is not incentivised to take a too high fee, as this would render completion of the transfer unlikely.
```

Comment:
I was not aware of any current `fee` discussions for the spec & implementation, so I cannot review this.
@LefterisJP ?